### PR TITLE
[llm] Use lower default learning rate and rename llm decoders to extractors

### PR DESCRIPTION
**Pull request by arnavgarg1**

### CODE REVIEW

File: ludwig/schema/trainer.py [hunk 1]
```diff
@@ -711,6 +711,21 @@ class GBMTrainerConfig(BaseTrainerConfig):
 class LLMTrainerConfig(BaseTrainerConfig):
     """Base class for all LLM trainer configs."""
 
+    learning_rate: Union[float, str] = schema_utils.OneOfOptionsField(
+        default=0.0001,
```
Comment (justinxzhao) [inline ludwig/schema/trainer.py:715]: Should this be even lower? Some of our BERT baseline configs use `0.00002`.

**Review by justinxzhao — APPROVED** [review]

**Review by tgaddair — APPROVED** [review]

File: ludwig/schema/trainer.py (hunk 1 above)
Comment (arnavgarg1) [reply ludwig/schema/trainer.py:715]: I think this is fine for now from what I've tried, we could go to maybe 0.00005 if we'd like
